kibbeh/src/ui/MessageElement.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { formatDistance, fromUnixTime } from "date-fns";
+
+import { SingleUser } from "./UserAvatar";
+
+export interface MessageElementProps {
+  user: {
+    username: string;
+    avatar: string;
+    isOnline: boolean;
+  };
+  msg: {
+    ts: number;
+    text: string;
+  };
+}
+
+interface MessageDateProps {
+  ts: number;
+}
+
+const MessageDate: React.FC<MessageDateProps> = ({ ts }) => (
+  <span
+    className="text-primary-300 font-medium inline-block truncate"
+    style={{
+      fontSize: "12px",
```

Comment:
You can use `text-sm` I think, but we'll need to override it in tailwind config to be `0.85rem`.